front/src/components/boxs/device-in-room/EditDeviceInRoom.jsx

```diff
@@ -13,7 +13,8 @@ import actions from '../../../actions/dashboard/edit-boxes/editDevicesInRoom';
 const SUPPORTED_FEATURE_TYPES = [
   DEVICE_FEATURE_TYPES.LIGHT.BINARY,
   DEVICE_FEATURE_TYPES.LIGHT.COLOR,
-  DEVICE_FEATURE_TYPES.LIGHT.BRIGHTNESS
+  DEVICE_FEATURE_TYPES.TELEVISION.CHANNEL,
```

Comment:
Not sure if you want to include it but televisions also have a source (HDMI1, HDMI2, TV etc)  in addition to a channel